frontend/src/metabase/visualizations/visualizations/LinkViz/EntityDisplay.tsx

```diff
@@ -50,3 +45,27 @@ export const EntityDisplay = ({
     </EntityDisplayContainer>
   );
 };
+
+export const UrlLinkDisplay = ({ url }: { url?: string }) => {
+  const urlIcon = isEmpty(url) ? "question" : "link";
```

Comment:
Not sure what's going on here; I assume we don't have a URL if a link points to a Metabase entity. If so, I wonder why we are using a `question` icon in that case. I believe we can also add links for models and dashboards?

Reply:
There's a separeate component for entity links, which work differently. If this component is rendered, it's either an external URL, or an empty state.

Reply:
Oh, gotcha, thanks for explaining! Nit: maybe extracting to a variable like `EMPTY_STATE_ICON` would make it a bit easier to parse. Definitely optional tho